lib/declarations.d.ts

```diff
@@ -317,8 +318,27 @@ interface IResourceLoader {
 	getPathToAppResources(framework: string): string;
 }
 
+/**
+ * Used to download resources from the server
+ */
 interface IResourceDownloader {
+	/**
+	 * Download all cordova javascript fies - Android, iOS and WP8
```

Comment:
Hm, what about NativeScript? (This might be an irrelevant comment. Ignore if it is.)